support-frontend/assets/helpers/redux/checkout/product/reducer.ts

```diff
@@ -52,10 +59,17 @@ export const productSlice = createSlice({
 		setOrderIsAGift(state, action: PayloadAction<boolean>) {
 			state.orderIsAGift = action.payload;
 		},
-		setDiscountedPrice(state, action: PayloadAction<ProductPrice>) {
-			state.discountedProductPrice = action.payload;
+		setStartDate(state, action: PayloadAction<DateYMDString>) {
+			state.startDate = action.payload;
 		},
 	},
+	extraReducers: (builder) => {
+		builder.addCase(setDeliveryCountry, (state, action) => {
+			if (state.productType === GuardianWeekly) {
+				state.fulfilmentOption = getWeeklyFulfilmentOption(action.payload);
+			}
+		});
+	},
```

Comment:
This is cool! So can we remove the address side effect now? https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/redux/checkout/address/sideEffects.ts#L43

Reply:
We'll still need it to be able to reset the payment method when the country changes, but once we've migrated payments too then yes- I've left a comment by the `'ON_DELIVERY_COUNTRY_CHANGED'` case in the old reducer.

Reply:
Ahhh yeah I've just got to that comment! Sounds good 👍